backend/src/kitconcept/intranet/services/feedback_contact_form.py

```diff
@@ -0,0 +1,293 @@
+from datetime import datetime
+from email.message import EmailMessage
+from email.utils import formataddr
+from kitconcept.intranet import _
+from kitconcept.intranet.controlpanels.intranet import IIntranetSettings
+from plone import api
+from plone.app.uuid.utils import uuidToObject
+from plone.registry.interfaces import IRegistry
+from plone.restapi.deserializer import json_body
+from plone.restapi.interfaces import IExpandableElement
+from plone.restapi.services import Service
+from Products.CMFCore.utils import getToolByName
+from Products.CMFPlone.interfaces.controlpanel import IMailSchema
+from zExceptions import BadRequest
+from zope.component import getMultiAdapter
+from zope.component import getUtility
+from zope.i18n import translate
+
+import logging
+
+
+logger = logging.getLogger("kitconcept.intranet")
+
+
+def get_registry_value(interface, field_name, prefix=None):
+    """Generic helper to fetch a registry field from a control panel schema."""
```

Comment:
Use the existing API (https://6.docs.plone.org/plone.api/portal.html#portal-get-registry-record-example) instead of recreating it.

Reply:
I wasn't able to make this work, so I kept my own implementation.

Reply:
@iFlameing Fixed in https://github.com/kitconcept/kitconcept.intranet/pull/170/commits/6ac26b43a2a75c277328d3240d2ebd9a5cab22b4